dump/sbe_dump_collector.cpp

```diff
@@ -433,4 +443,18 @@ bool SbeDumpCollector::executeThreadStop(struct pdbg_target* target)
     return true;
 }
 
+void SbeDumpCollector::addLogDataToDump(uint32_t pelId, std::string src,
+                                        std::string chipName, uint64_t chipPos,
+                                        const std::filesystem::path& path)
+{
+    std::filesystem::path info = path / "info.yaml";
+    std::ofstream fout(info);
+
+    fout << "ErrorInfo:" << std::endl;
```

Comment:
Suggestion : This fout could fail. I am not clear how we handle this failure in other places across.  So please handle accordingly.